### PR TITLE
[Anomaly Task] 🐞 Fix inference when model backbone changes

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
# Description

- Currently anomaly tasks fail when the model backbone changes. This fix checks for the value of backbone in requested model and the backbone in the new model. It then creates model with the old backbone for inference.

# Linting checks

image

# 🧪

- http://validationreports.sclab.intel.com:8006/reports/build_number_report?test_session_build_number=pr-ote-test-571&environment=iotg-dev-workstation-08
- BMM- https://10.211.120.59/projects/63175d4d8d2bfdf63639e8b1/models

### CODE REVIEW

**Comment by ashwinvaidya17** [comment]
run ote-test

**Review by djdameln — COMMENTED** [review]
Did you confirm that the new backbone is used in the next training round?

File: external/anomaly/configs/base/padim/configuration.py [hunk 1]
```diff
@@ -38,10 +38,14 @@ class LearningParameters(BaseAnomalyConfig.LearningParameters):
         header = string_attribute("Learning Parameters")
         description = header
 
+        # Editable is set to false as wide_resent_model is very large for
```
Comment (samet-akcay) [inline external/anomaly/configs/base/padim/configuration.py]: Wide ResNet model or WideResNet

**Review by samet-akcay — COMMENTED** [review]
looking good to me

**Review by djdameln — APPROVED** [review]

**Comment by ashwinvaidya17** [comment]
> Did you confirm that the new backbone is used in the next training round?

I just checked if it uses the new backbone for the next training round. I tried optimization and hpo and it works fine.

**Review by samet-akcay — APPROVED** [review]

**Pull request merged**